packages/jumpstarter-driver-tftp/jumpstarter_driver_tftp/client.py

```diff
@@ -108,3 +104,10 @@ def get_port(self) -> int:
             int: The port number (default is 69)
         """
         return self.call("get_port")
+
+    def _compute_checksum(self, operator: Operator, path: str) -> str:
+        hasher = hashlib.sha256()
+        with operator.open(path, "rb") as f:
+            while chunk := f.read(8192):
```

Comment:
```suggestion
            while chunk := f.read(8192):
```

Probably even go for bigger chunks, it will work much quicker. 1MB, 4MB ... etc.

Can we add some debug logging before, after, printing "we are starting the checksum of  blahblah", "the checksum of blahblah is xxxxxxxxxxxxxxxx"

In such way we can identify how long does it take to compute in debug mode.